example/tests/conftest.py

```diff
@@ -62,3 +62,13 @@ def single_art_project(art_project_factory):
 @pytest.fixture
 def client():
     return APIClient()
+
+
+@pytest.fixture
+def blog():
```

Comment:
This is actually not needed as this is automatically done with pytest-factoryboy where this factory is [registered](https://github.com/django-json-api/django-rest-framework-json-api/pull/490/files#diff-b9e1915dc58c0e683db3b4b7a7a7c365R18).

Reply:
Oh that's my bad on not seeing the registered factories.